### PR TITLE
PrimeVideo regex

### PR DESCRIPTION
**Pull request by Bas950**
Just ignore the changes in the amazon presence.ts, as it only adds comments. (There were changes done to the file and to merge it together I had to commit.)

### CODE REVIEW

File: Prime Video/dist/metadata.json [hunk 1]
```diff
@@ -8,30 +8,12 @@
...
-    "www.amazon.es",
-    "www.amazon.ca",
-    "www.amazon.mx",
-    "www.amazon.com",
-    "www.amazon.com.au",
-    "www.amazon.com.br",
-    "www.primevideo.com"
-  ],
-  "version": "1.0",
+  "url": "www.primevideo.com",
```
Comment (MrSpinne) [inline Prime Video/dist/metadata.json]: Seems unnecessary (the url) If using Regex

**Comment by MrSpinne** [comment]
Could you please change
`regex = RegExp("https:\\/\\/www\\.amazon\\.(.*?)\\/\\b(?:Prime-Video|Prime-Instant-Video|gp\\/video)\\b");`

to

`regex = RegExp("https:\\/\\/www\\.amazon\\.(.*?)\\/\\b(?:Prime-Video|Prime-Instant-Video|Amazon-Video|gp\\/video)\\b");`

**Comment by Bas950** [comment]
> Could you please change
> `regex = RegExp("https:\\/\\/www\\.amazon\\.(.*?)\\/\\b(?:Prime-Video|Prime-Instant-Video|gp\\/video)\\b");`
> 
> to
> 
> `regex = RegExp("https:\\/\\/www\\.amazon\\.(.*?)\\/\\b(?:Prime-Video|Prime-Instant-Video|Amazon-Video|gp\\/video)\\b");`

I will DM you on discord xD

File: Prime Video/dist/metadata.json (hunk 1 above)
Comment (Bas950) [reply Prime Video/dist/metadata.json]: you need a default url if for some reason the regex gives out an error

Comment (MrSpinne) [reply Prime Video/dist/metadata.json]: Interesting

**Pull request merged**